gdcdictionary/schemas/_terms.yaml

```diff
@@ -231,6 +303,50 @@ fastq_name: # TOREVIEW
   description: >
     Names of FASTQs.
 
+fev1_ref_post_bronch_%:
```

Comment:
I'm not a huge fan of having special characters in property names, anyway we could avoid the `%`?